app/mailers/vaccination_center_mailer.rb

```diff
@@ -0,0 +1,15 @@
+class VaccinationCenterMailer < ApplicationMailer
+  default from: "Covidliste <partenaires@covidliste.com>"
+
+  def confirmed_vaccination_center_instructions
+    @vaccination_center = params[:vaccination_center]
+    partner_email = @vaccination_center&.partners&.first&.email
+
+    return if @vaccination_center.partners.empty?
```

Comment:
Au dessus tu fais des &, ca veut dire que tu t'attends à ce que l'email final puisse être nil, et donc si il ne l'est pas c'est que le centre a un partenaire et voilà son email
```suggestion
    return if partner_email
```